src/AutoRest.CoreShared/Program.cs

```diff
@@ -46,7 +44,6 @@ private static void Main()
                 .Replace("options.GetType().Namespace", "options.GetType().Namespace!");
             File.WriteAllText($"../../{Path}/ClientDiagnostics.cs", cleanFile);
             File.WriteAllText($"../../AutoRest.CSharp.V3/Azure.Core.Shared/ClientDiagnostics.cs", cleanFile);
-            File.WriteAllText($"../../../test/AutoRest.TestServer.Tests/Azure.Core.Shared/ClientDiagnostics.cs", cleanFile);
```

Comment:
These no longer exist since the test project uses references to the source from the `assets` folder.